src/components/Layout/Layout.js

```diff
@@ -41,6 +42,8 @@ function Layout(props) {
   const location = useLocation();
   const locationPrefix = smartLocationPrefix(location);
 
+  const HomepageRoute = [{ path: "/homepage", element: <Homepage /> }];
```

Comment:
Homepage needs to run on `/`. That is `aave.blockanalitica.com` and not on any subpath.
Instead of doing this with all the homepage route thingies, you could just change the line that redirects `/` to the `/v2/ethereum/` from
`<Route index element={<Navigate replace to="/v2/ethereum/" />} />`
to
`<Route index element={<Homepage>} />` and it should work correctly afaik